src/main/java/com/smoothstack/lms/adminmicroservice/controller/AdminController.java

```diff
@@ -363,11 +393,11 @@
 
 	@PutMapping(path = "administrator/branches/{id}")
 	@Consumes({ MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE })
-	public ResponseEntity<Void> updateBranch(@RequestBody Branch branch, @PathVariable int id) {
-		if (branch == null || branch.getBranchName() == null || branch.getBranchAddress() == null) {
+	public ResponseEntity<Void> updateLibraryBranch(@RequestBody LibraryBranch branch, @PathVariable Long id) {
+		if (branch == null || branch.getName() == null || branch.getAddress() == null) {
 			return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
 		}
-		branch.setBranchId(id);
+		branch.setId(id);
 		try {
 			adminService.updateBranch(branch);
 			return new ResponseEntity<>(HttpStatus.OK);
```

Comment:
This try catch pattern is redundant, you're essentially just doing what the jvm was going to do for you anyway. You can remove the try catch block and make sure just not to suppress any exceptions raised in the service call.